collection-scripts/gather_etcd

```diff
@@ -36,6 +36,34 @@ set +o errexit
 
 PIDS=()
 
+# Optional route to gather additional etcd information, including metrics.
+if [ $# -ge 1 ] && [ "$1" = "gather-more-etcd" ]; then
```

Comment:
Would it make sense creating a new script file (e.g. `gather_etcd_additional` or something like that) instead of maintaining this separate block inside the `gather_etcd`?.

So that users just need to call `oc adm must-gather -- gather_etcd_more ...`?